### PR TITLE
Canon next 0.1.5

### PR DESCRIPTION
**Pull request by scespinoza**
- Adds custom sections support.
- Improves ProfileSearch component

Update: 0.1.6 pushes back on custom sections. Working on the monorepo, but not on the monshaat project, its a breaking change so I had to roll back.

### CODE REVIEW

File: example-next/pages/profile/[...members].js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import {useRouter} from "next/router";
 import {serverSideTranslations} from "next-i18next/serverSideTranslations";
 import {Profile, NonIdealState, cmsDefaultPaths, cmsDefaultProps} from "@datawheel/canon-next";
 
+
```
Comment (davelandry) [inline example-next/pages/profile/[...members].js:14]: ...if I'm not merging right away, should we remove this line? 😏

**Review by davelandry — CHANGES_REQUESTED** [review]
@scespinoza this is looking great, and it makes sense to me. I see what you mean about commenting out the customSections for now. Are you intending for me to merge/release this so we can get the ProfileSearch improvements, or do you want to keep working in here until you get CustomSections working? (if so, I'd also like you to add something to the README describing the folder/config settings needed!)

**Comment by bidwolf** [comment]
Hey guys, Good morning!
I'm working to update a new version of a Brazilian website: codelife.com that depends on canon's old versions to work.
I would like to maintain this partnership, I would love it if it were possible to integrate this framework with next-js and the new versions of react, so if I can help in any way I'm open to that.

**Comment by davelandry** [comment]
hi @bidwolf! I just published a version of an internal guide we wrote to start migrating projects to Nextjs (and completely abandoning canon-core). Check it out, and if you have any questions please make a new Github issue in this repository and we can discuss/help!

https://github.com/Datawheel/canon/wiki/Migrating-from-canon-to-Next.js

**Review by davelandry — APPROVED** [review]

**Pull request merged**